pages/linux/csplit.md

```diff
@@ -0,0 +1,19 @@
+# csplit
+
+> Split a file into pieces.
+
+- Split a file at lines 5 and 23:
+
+`csplit {{file}} 5 23`
+
+- Split a file every 5 lines (-k flag necessary if not divisible by 5):
```

Comment:
We can have a separate example showing the importance of `-k` and not mix it with this example.

Reply:
I added an example in cd8321e15e8 but I'm not sure of the wording, it feels a bit clunky.